### PR TITLE
jasper: Bump dependencies + update imports

### PR DESCRIPTION
**Pull request by prince-chrismc**
Needed for #13988 and #14281

Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/README.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
Duplicate of https://github.com/conan-io/conan-center-index/pull/13285, which requires https://github.com/conan-io/conan-center-index/issues/13159

**Comment by prince-chrismc** [comment]
> Duplicate of https://github.com/conan-io/conan-center-index/pull/13285, which requires https://github.com/conan-io/conan-center-index/issues/13159

but i dont want the new version just a dep bump 😕

**Comment by HappySeaFox** [comment]
> > Duplicate of #13285, which requires #13159
> 
> but i dont want the new version just a dep bump 😕

It seems jasper cannot be bumped because of the Windows SDK issue.

**Comment by SpaceIm** [comment]
Yes, still the same issue for 2 months https://github.com/conan-io/conan-center-index/issues/13159

**Comment by prince-chrismc** [comment]
Its only one compiler that has a problem - its not a blocker - lets keep moving things forwards.
We should be trying to progress and improve things as a much as possible.

There's a tiny % that's currently impacted look at the august number so I would not stop the rest of the world.

File: recipes/jasper/all/conanfile.py [hunk 1]
```diff
@@ -38,27 +42,30 @@ def _build_subfolder(self):
...
     def requirements(self):
         if self.options.with_libjpeg == "libjpeg-turbo":
             self.requires("libjpeg-turbo/2.1.2")
         elif self.options.with_libjpeg == "libjpeg":
-            self.requires("libjpeg/9d")
+            self.requires("libjpeg/9e")
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) >= "16":
+            raise ConanInvalidConfiguration(f"{self.name} Current can not build in CCI due to windows SDK version. See https://github.com/conan-io/conan-center-index/pull/13285 for the solution hopefully")
```
Comment (SpaceIm) [inline recipes/jasper/all/conanfile.py:65]: I don't think it's acceptable, it has too many consequences in downstream recipes.

Comment (HappySeaFox) [reply recipes/jasper/all/conanfile.py:65]: Agree. The problem is just in the CI configuration. If a user has the latest SDK installed on a local machine, which is highly possible, the issue will not raise locally.

Comment (prince-chrismc) [reply recipes/jasper/all/conanfile.py:65]: Thanks for your input.

This is not new and we do it in tons of recipes.
You can always host your own copies of the recipes while you wait for this issue to be resolved.

Happy contributing!

File: recipes/jasper/all/conanfile.py [hunk 2]
```diff
@@ -38,27 +42,30 @@ def _build_subfolder(self):
...
 
     def requirements(self):
         if self.options.with_libjpeg == "libjpeg-turbo":
             self.requires("libjpeg-turbo/2.1.2")
         elif self.options.with_libjpeg == "libjpeg":
-            self.requires("libjpeg/9d")
+            self.requires("libjpeg/9e")
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) >= "16":
```
Comment (SpaceIm) [inline recipes/jasper/all/conanfile.py:64]:
```suggestion
        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) == "16":
```

I disagree with this strategy, but at least please don't raise for Visual Studio 2022.

**Review by uilianries — APPROVED** [review]
LGTM

**Review by ericLemanissier — APPROVED** [review]

**Pull request merged**